src/components/displayBook.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import '../App.css';
+import { useSelector } from 'react-redux';
+import Display from './display';
+
+const DisplayBook = () => {
+  const books = useSelector((state) => state.booksReducer);
```

Comment:
- [ ] OPTIONAL - Also consider changing this component name from DisplayBook to simply `Books` since it rendering a collection of `Book` component.